rfd/0105-headless-authentication.md

```diff
@@ -153,7 +153,7 @@ A request id will be derived from the client's public key so that an attacker ca
 
 Note: We could also use the public key directly (base64 encoded), but we choose to use a UUID to shorten the URL and improve its readability.
 
-As [explained above](#unauthenticated-headless-login-endpoint), the Auth server will write the request details to the backend under `/headless_authentication/<request_id>` on demand. It will have a 1 minute TTL, by which point the user should have completed the headless authentication flow. The request will begin in the pending state. The Auth server then waits for the user to approve the authentication request using a resource watcher.
+As [explained above](#unauthenticated-headless-login-endpoint), the Auth server will write the request details to the backend under `/headless_authentication/<request_id>` on demand. It will have a 3 minute TTL, matching the callback timeout of the request. The request will begin in the pending state. The Auth server then waits for the user to approve the authentication request using a resource watcher.
```

Comment:
I'd be less specific here. Every single time we change the timeout, we would need to update the RFD. I think that "a few minutes" or "short TTL" should be fine.